sdk/inkrecognizer/README.md

```diff
@@ -0,0 +1,50 @@
+# Azure InkRecognizer library for JavaScript
+
+- [API Reference documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/ink-recognizer/)
+
+## Introduction
+
+This project provides a SDK in Javascript that makes it easy to consume Microsoft Azure InkRecognizer service.
+
+## Building the library
+
+Once you clone this repo, you can build the package with the following commands:
+
+```sh
+npm install
+npm run build
+```
+
+Run tests via:
+
+```sh
+npm test
+```
+
+## Implementation Details
+
+The overall build pipeline looks like the following:
+
+1. TypeScript builds all source files under `./src` to ECMAScript Modules (ESM) under `./dist-esm`
+2. Rollup builds `./dist-esm` to an optimized single file at `./dist/index.js` as the Node entry point.
+3. Rollup builds `./dist-esm` to an optimized browser bundle under `./browser/index.js`.
+
+Tests follow a similar pipeline, however output folders have the `test-` prefix.
+
+## License
+
+This project is licensed under MIT.
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a
```

Comment:
The [contributing guidelines](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) are probably a more comprehensible reference 🌞 please use them if they're sufficient, otherwise we should probably submit an issue and update the guidelines.

Reply:
Yep, this seems better. I will remove the Contributing section here to respect the global one.